diff --git a/build/top.mk b/build/top.mk
index 6850121..283fe20 100644
--- a/build/top.mk
+++ b/build/top.mk
@@ -170,8 +170,8 @@ LDAP_LIBREWRITE_A = $(LDAP_LIBDIR)/librewrite/librewrite.a
 LDAP_LIBLUNICODE_A = $(LDAP_LIBDIR)/liblunicode/liblunicode.a
 LDAP_LIBLUTIL_A = $(LDAP_LIBDIR)/liblutil/liblutil.a
 
-LDAP_L = $(LDAP_LIBLUTIL_A) \
-	$(LDAP_LIBLDAP_LA) $(LDAP_LIBLBER_LA)
+LDAP_L = $(LDAP_LIBLDAP_LA) $(LDAP_LIBLUTIL_A) \
+	$(LDAP_LIBLBER_LA)
 SLAPD_L = $(LDAP_LIBLUNICODE_A) $(LDAP_LIBREWRITE_A) \
 	$(LDAP_LIBLUTIL_A) $(LDAP_LIBLDAP_R_LA) $(LDAP_LIBLBER_LA)
 
diff --git a/tests/progs/Makefile.in b/tests/progs/Makefile.in
index 8904190..2d4cf43 100644
--- a/tests/progs/Makefile.in
+++ b/tests/progs/Makefile.in
@@ -24,7 +24,7 @@ SRCS     = slapd-common.c \
 LDAP_INCDIR= ../../include
 LDAP_LIBDIR= ../../libraries
 
-XLIBS    = $(LDAP_LIBLDAP_LA) $(LDAP_LIBLUTIL_A) $(LDAP_LIBLBER_LA)
+XLIBS    = $(LDAP_LIBLUTIL_A) $(LDAP_LIBLDAP_LA) $(LDAP_LIBLBER_LA)
 XRLIBS    = $(LDAP_LIBLDAP_R_LA) $(LDAP_LIBLUTIL_A) $(LDAP_LIBLBER_LA)
 XXLIBS	 = $(SECURITY_LIBS) $(LUTIL_LIBS)
 RLIBS = $(XRLIBS) $(XXLIBS) $(AC_LIBS) $(XXXLIBS)
